doc/wiki.txt

```diff
@@ -167,29 +167,27 @@ GUIDE                                                        *wiki-intro-guide*
 
 Executive summary~
 * Speed is key - make sure you have a fast workflow.
-  * Use a global keyboard shortcut to open the wiki index, e.g. <alt-n>.
+  * Use a global keyboard shortcut to map |:WikiIndex| to open the wiki index,
+    e.g. <alt-n>.
   * Use |:WikiFzfPages| and `ripgrep` (with e.g. ctrlsf.vim) for searching.
 * Write short pages and use links between related pages.
 * Don't obsess about the wiki structure.
 
-
```

Comment:
The idea behind this and the next extra line spacing is to set off the "preface" text. The real guide/essay starts at "People write things down...". Perhaps I should set it off in a different manner?

Reply:
Oh, I thought it was forgotten. Maybe use some kind of separator?

Reply:
I'll fix in an update.

Reply:
I'm reverting for now.

Reply:
9af22a3fa3694ce186bbcdb2b2017a4ff2fca956